src/storage/src/healthcheck.rs

```diff
@@ -27,28 +26,12 @@ pub async fn write_to_persist(
     status_shard: ShardId,
 ) {
     let now_ms = now();
-    let row = {
-        let timestamp = NaiveDateTime::from_timestamp_opt(
-            (now_ms / 1000)
-                .try_into()
-                .expect("timestamp seconds does not fit into i64"),
-            (now_ms % 1000 * 1_000_000)
-                .try_into()
-                .expect("timestamp millis does not fit into a u32"),
-        )
-        .unwrap();
-        let timestamp = Datum::TimestampTz(
-            DateTime::from_utc(timestamp, Utc)
-                .try_into()
-                .expect("must fit"),
-        );
-        let collection_id = collection_id.to_string();
-        let collection_id = Datum::String(&collection_id);
-        let status = Datum::String(new_status);
-        let error = new_error.into();
-        let metadata = Datum::Null;
-        Row::pack_slice(&[timestamp, collection_id, status, error, metadata])
-    };
+    let row = mz_storage_client::healthcheck::pack_status_row(
```

Comment:
For your convenience, the definition of pack_status_row:
https://github.com/MaterializeInc/materialize/blob/2913d565aa5e444420a17a94997152d76fa8eade/src/storage-client/src/healthcheck.rs#L19-L38